### ISSUE TITLE
Pass Play vs Scramble

### ISSUE THREAD

**Issue by cminton** [BurntSushi/nflgame#215]
Is there a way to tell the difference between a QB scramble (a pass play which turns into a rush play) versus a designed QB rush play?

**Comment by BurntSushi** [comment]
I don't think so. You might consider looking at the human readable description of some of the plays and see if you can develop any heuristics for inferring the type of play from that.

**Comment by cminton** [comment]
Thanks Andrew. I'll take a look.
